torchaudio/functional/functional.py

```diff
@@ -135,6 +136,95 @@ def spectrogram(
     return spec_f
 
 
+def inverse_spectrogram(
+        spectrogram: Tensor,
+        length: Optional[int],
+        pad: int,
+        window: Tensor,
+        n_fft: int,
+        hop_length: int,
+        win_length: int,
+        normalized: bool,
+        center: bool = True,
+        pad_mode: str = "reflect",
+        onesided: bool = True,
+        return_complex: bool = False,
+) -> Tensor:
+    r"""Create an inverse spectrogram or a batch of inverse spectrograms from the provided
+    complex-valued spectrogram.
+
+    Args:
+        spectrogram (Tensor): Complex tensor of audio of dimension (..., freq, time).
+            Alternatively, the tensor can be pseudo-complex, i.e. a real tensor of
+            dimension (..., freq, time, 2), if return_complex is False, although this
+            behavior is deprecated.
+        length (int): The amount to trim the signal by (i.e. the original signal length).
+            Default: None (whole signal).
+        pad (int): Two sided padding of signal
+        window (Tensor): Window tensor that is applied/multiplied to each frame/window
+        n_fft (int): Size of FFT
+        hop_length (int): Length of hop between STFT windows
+        win_length (int): Window size
+        normalized (bool): Whether the stft output was normalized by magnitude
+        center (bool, optional): whether the waveform was padded on both sides so
+            that the :math:`t`-th frame is centered at time :math:`t \times \text{hop\_length}`.
+            Default: ``True``
+        pad_mode (string, optional): controls the padding method used when
+            :attr:`center` is ``True``. This parameter is provided for compatibility with the
+            spectrogram function and is not used. Default: ``"reflect"``
+        onesided (bool, optional): controls whether spectrogram was done in onesided mode.
+            Default: ``True``
+        return_complex (bool, optional):
```

Comment:
I think we can get rid of `return_complex`, for the same reason as we are removing the `power` argument. We cannot achieve the complete symmetry in function arguments, and complex-ness can be detected at run time.